docs/providers-configurations-ref.rst

```diff
@@ -0,0 +1,99 @@
+ .. Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+ ..   http://www.apache.org/licenses/LICENSE-2.0
+
+ .. Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+.. jinja:: config_ctx
+
+    Configuration Reference
+    .......................
+
+    This page contains the list of all available Airflow configurations for the
+    ``{{ package_name }}`` provider that can be set in ``airflow.cfg`` file or using environment variables.
+
+    .. note::
+        For more information on setting the configuration, see :doc:`apache-airflow:howto/set-config`
```

Comment:
```suggestion
        For more information see :doc:`apache-airflow:howto/set-config`.
```

I think this would render better: "For more information see <Setting Configuration Options>."